README.rst

```diff
@@ -1113,7 +1113,7 @@ Choose from the list of available rules:
 
   Replaces superfluous ``elseif`` with ``if``.
 
-* **no_superfluous_phpdoc_tags**
+* **no_superfluous_phpdoc_tags** [@Symfony, @PhpCsFixer]
```

Comment:
Do we want to adopt this here? In general the `@PhpCsFixer` ruleset is a superset of `@symfony`. But there would be many changes in this repo.
So what do you think about this rule? (I like it!)

Reply:
:+1: